tests/rptest/services/redpanda.py

```diff
@@ -84,6 +86,14 @@
     'MetricSample', ['family', 'sample', 'node', 'value', 'labels'])
 
 
+@dataclass
+class UsageStats:
```

Comment:
It would be nice to see a "self test" for these, e.g., a small test which does something which increments these counts then checks they are in the expected range.